behavior_metrics/utils/script_manager_gazebo.py

```diff
@@ -22,7 +22,9 @@
 import rospy
 import random
 import sys
-
+import matplotlib.pyplot as plt
+import mplcursors
```

Comment:
```
Traceback (most recent call last):
  File "driver_gazebo.py", line 24, in <module>
    from utils import environment, script_manager_gazebo
  File "/home/jderobot/Documents/Projects/BehaviorMetrics/behavior_metrics/utils/script_manager_gazebo.py", line 26, in <module>
    import mplcursors
ModuleNotFoundError: No module named 'mplcursors'
```

It throws this error trying to run:
`python3 driver_gazebo.py -c configs/default-rl-qlearn.yml -g`